core/src/providers/openai.rs

```diff
@@ -121,7 +121,7 @@ pub struct OpenAIToolFunction {
 }
```

Comment:
Diff is broken, I only removed `chat_completion` and `streamed_chat_completion` and rename the one with `_tools`.